frontend/src/app/components/block-sizes-weights-graph/block-sizes-weights-graph.component.ts

```diff
@@ -124,6 +125,7 @@ export class BlockSizesWeightsGraphComponent implements OnInit {
       color: [
         '#FDD835',
         '#D81B60',
+        '#14EDF5',
```

Comment:
For the color, I'd try `#039BE5` so it's less flashy. For future reference you can see the colors we usually pick for charts in https://github.com/mempool/mempool/blob/master/frontend/src/app/app.constants.ts#L34

```suggestion
        '#039BE5,
```